### PR TITLE
Erichs changes

### PR DESCRIPTION
**Pull request by thalaeg**
### Summary

These are fixes for textual errors in the documentation.

### Checklist:
- [ ] Documentation NA
- [x] Spellchecked my updates
- [x] Ready to be merged 

You'll notice some reverts. This is in reference to an issue that I tried to fix, but could quite do so I reverted them. Only the textual changes/improvements are here.

### CODE REVIEW

File: app/_hub/heroku/kong-app/index.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ type: integration
 desc: Deploy Kong CE clusters to Heroku Common Runtime or Private Spaces
 
 description: |
-  Easily deploy Kong Community Edition as a Heroku app. Supports both
+  Easily deploy Kong Community Edition as a Heroku app. Support for both
```
Comment (coopr) [inline app/_hub/heroku/kong-app/index.md]: The prior grammar was correct.

File: app/_hub/kong-inc/request-transformer-advanced/index.md [hunk 2]
```diff
@@ -136,7 +136,7 @@ params:
 
 ### Template as Value
 
-User can use any of the the current request headers, query params, and captured URI named groups as template to populate above supported config fields.
+Users can use any of the the current request headers, query params, and captured URI named groups as template to populate above supported config fields.
```
Comment (coopr) [inline app/_hub/kong-inc/request-transformer-advanced/index.md]: Please make this improvement to prior versions of this documentation, eg. https://github.com/Kong/docs.konghq.com/blob/master/app/_hub/kong-inc/request-transformer-advanced/0.32-x.md

File: app/_hub/kong-inc/request-transformer-advanced/index.md [hunk 3]
```diff
@@ -163,7 +163,7 @@ $ curl -X POST http://localhost:8001/apis \
 
 Enable the ‘request-transformer-advanced’ plugin to add a new header `x-consumer-id`
 and its value is being set with the value sent with header `x-user-id` or
-with the default value alice is `header` is missing.
+with the default value alice `header` is missing.
```
Comment (coopr) [inline app/_hub/kong-inc/request-transformer-advanced/index.md]: Even with this change this sentence is still confusing - can you try to clarify it?

File: app/_hub/heroku/kong-app/index.md (hunk 1 above)
Comment (levfishbluefish) [inline app/_hub/heroku/kong-app/index.md]:
```suggestion
  Easily deploy Kong Community Edition as a Heroku app. Supports both
```

**Pull request closed without merging**